src/sentry/api/serializers/models/rule.py

```diff
@@ -195,4 +197,12 @@ def serialize(self, obj, attrs, user, **kwargs):
         else:
             d["snooze"] = False
 
+        org = Organization.objects.get(id=obj.project.organization_id)
+        try:
+            neglected_rule = NeglectedRule.objects.get(rule=obj, organization=org, opted_out=False)
```

Comment:
@ceorourke Nit: you can do `organization_id=obj.project.organization_id`

Reply:
I need to pass the org though, not the ID

Reply:
I guess since I'm not looking anything up through the neglected rule object to org attributes it's ok? Just seems like the wrong way to look stuff up

Reply:
@ceorourke what seems like the wrong way? Using the `id` instead of the whole org? I think it's fine if it saves a query.